worker-document-interface/src/main/java/com/hpe/caf/worker/document/model/ResponseOptions.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.worker.document.model;
+
+import java.util.Map;
+
+/**
+ * Class containing additional worker response properties.
+ * This class is immutable.
+ */
+public class ResponseOptions
```

Comment:
- Sorry but this should be an interface and if it wants to be in this package with all the other model objects then it needs to extend `DocumentWorkerObject`.
- Sorry but it should also have a `getTask()` to get back to the task that it is a response for.
- My first impression is that we should just call this `Response`.  I'm not sure that the `Options` suffix is really adding anything.

Reply:
Re naming - Response vs ResponseOptions - IMO Response name is too confusing. This class doesn't have full Response data. It only has (additional) options for the response.

Reply:
We may add more stuff to it in the future about how the response is formed.